R/SparkR/sparkrMosaic/R/enableMosaic.R

```diff
@@ -0,0 +1,26 @@
+#' enableMosaic
```

Comment:
This file is a duplicate of R/SparkR/enableMosaic.R

Do we need both?

Reply:
"maybe" would be the short answer. We need the version in `R/SparkR/sparkrMosaic/R/enableMosaic.R` for the file to get packaged up.

When I was writing this, my intention was that everything in the `sparkrMosaic` folder would be automatically created by the build - this is why the file from `R/SparkR/enableMosaic.R` gets copied across. My thinking at the time was that this would simplify updates as you'd only need need to update files in one place if we autogenerated all the code. That's still the ambition, but as per the tests comment above it didn't pan out yet!

Reply:
Resolved by auto-generating all the sparkrMosaic